src/System.ComponentModel.EventBasedAsync/tests/System.ComponentModel.EventBasedAsync.Tests.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <!-- this tests run in multiple AppDomains in full framework, so if an Assert fails we would get a Serialization exception instead of the real failure.
     adding this xunit so it runs in one AppDomain -->
-    <XunitOptions Condition="'$(TestTFM)'=='net46'">$(XunitOptions) -noappdomain </XunitOptions>
+    <XunitDisableAppDomain>true</XunitDisableAppDomain>
```

Comment:
>XunitDisableAppDomain [](start = 5, length = 21)

nit: I am not good in naming but maybe XUnitNoAppdomain may be better as it match what we pass to Xunit runner